knowledge/science/botany/dianthus_caryophyllus/qna.yaml

```diff
@@ -0,0 +1,29 @@
+created_by: aashenfe
+domain: flowers
+seed_examples:
+- answer: |
+    The name "carnation" is believed to come 
+    from the Latin corona-ae, a "wreath, garland, 
+    chaplet, crown"
+  question: How did it get it's name
```

Comment:
Too ambiguous of a question, when writing the qna files, you can assume the model has no context. Try "how did the carnation flower get its name"